examples/tutorials/Part 12 - Train an Encrypted Neural Network on Encrypted Data.ipynb

```diff
@@ -52,10 +55,23 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# Create our dataset\n",
+    "# A Toy Dataset\n",
+    "data = torch.tensor([[0,0],[0,1],[1,0],[1,1.]])\n",
+    "target = torch.tensor([[0],[0],[1],[1.]])\n",
     "\n",
-    "data = torch.Tensor([[0., 0.], [0., 1.], [1., 0.], [1., 1.]])\n",
-    "target = torch.Tensor([[0.], [0.], [1.], [1.]])"
+    "# A Toy Model\n",
+    "model = nn.Linear(2,1)"
```

Comment:
Does it work for a model with multiple layers?

Reply:
I didn't had the time to try it for now :)

Reply:
Would recommend trying with multiple layers (even if they're multiple linear layers) eg something like
```
class Net(nn.Module):

    def __init__(self):
        self.l1 = nn.linear(2,2)
        self.l2 = nn.linear(2,1)
    def forward(self, input):
        hidden = self.l1(input)
        return self.l2(hidden)
```

Reply:
Ok I can include this in this PR

Reply:
I've added an example with this (and some activation), it works but I think there's an issue in the code because the loss doesn't decrease fast enough

Reply:
Actually it's all good